clr.defines.targets

```diff
@@ -2,6 +2,7 @@
     <!-- Features we're currently flighting, but don't intend to ship in officially supported releases -->
     <PropertyGroup Condition="'$(IsPrerelease)' == 'true'">
```

Comment:
Could you please make sure that this is not set in the Release branch? (It is not the case today.)

Reply:
It is still _true_ in the _release/3.0_ branch currently, but that's because there's not yet an officially shipping release coming from that branch. This was set correctly to _false_ in the _release/2.x_ branches (see https://github.com/dotnet/coreclr/commit/5b5dca3254e968de60524621c10fec3fd5dc3bd0).

Reply:
The previews shipping out of release/3.0 are proxies for officially shipping releases. We should flip that in release/3.0 now. It has not been flipped because of we do not have anything really depending on it for 3.0.

Reply:
I also kicked off a local build with `IsPrerelease=false` and ensured that no part of the feature inadvertently makes its way in to System.Private.CoreLib.dll.